### PR TITLE
Fixes #6037: Add multi-arch builder to ansible and helm plugins

### LINKED ISSUE #6037: Introduce Makefile target to build multi-arch solutions
#### Describe the problem you need a feature to resolve.
With the bump in KB, it would be useful to introduce the Makefile target to support building multiple architectures. 
Please refer:  https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/Makefile#L82-L97. This needs to be introduced in Ansible, Helm and Hybrid plugins. 

 /language ansible
 /language helm

### PR DESCRIPTION
**Pull request by jmrodri**
Signed-off-by: jesus m. rodriguez <jesusr@redhat.com>

**Description of the change:**
Similar to the go plugin, adding support for building multi-arch images from the project `Makefile`.

**Motivation for the change:**
Make it easier to build multi-arch operator images.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [X] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go [hunk 1]
```diff
@@ -102,6 +102,23 @@ docker-build: ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
```
Comment (laxmikantbpandhare) [inline internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go:105]: I guess we need test data as we updated the plugins.

**Review by laxmikantbpandhare — CHANGES_REQUESTED** [review]

File: internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go (hunk 1 above)
Comment (varshaprasad96) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go:105]: +1, testdata/ and docs would be helpful to users (as this was one of the frequently asked options by upstream community)

Comment (everettraven) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go:105]: +1

Comment (laxmikantbpandhare) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go:105]: testdata looks good to me.

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm for code changes and tesdata

**Pull request merged**